console/src/declaration.d.ts

```diff
@@ -6,3 +6,5 @@ declare module '*.scss' {
 declare namespace SvgPanZoom {
   export interface Instance {}
 }
+
+declare module 'react-bootstrap/lib/Tooltip';
```

Comment:
Can you just add the react-bootstrap from definitelyTyped?

Reply:
Yeah, I tried. But typings are not exported properly. Will investigate later.